apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -56,41 +74,86 @@ TODO: Fix asset loading in environments that don't use `pathPrefix`. This is a t
+| `mute`             | `() => void`                                       | Mutes the video.                                                                               |
+| `unmute`           | `() => void`                                       | Unmutes the video.                                                                             |
+| `toggleMute`       | `() => void`                                       | Toggles between muting and unmuting the video.                                                 |
+| `setDuration`      | `(duration: number) => void`                       | Sets the duration of the video.                                                                |
+| `seek`             | `(time: number) => void`                           | Seeks the video to a specific time in seconds.                                                 |
+| `seekToPercent`    | `(percent: number) => void`                        | Seeks the video to a specific percentage of its duration.                                      |
+| `seekRelative`     | `(secondsValOrFn: SetStateAction<number>) => void` | Seeks the video forward or backward by a given number of seconds.                              |
+| `enableCC`         | `() => void`                                       | Enables closed captions.                                                                       |
+| `disableCC`        | `() => void`                                       | Disables closed captions.                                                                      |
+| `toggleCC`         | `() => void`                                       | Toggles closed captions on and off.                                                            |
+| `enterFullScreen`  | `() => void`                                       | Enters fullscreen mode.                                                                        |
+| `exitFullScreen`   | `() => void`                                       | Exits fullscreen mode.                                                                         |
+| `toggleFullScreen` | `() => void`                                       | Toggles between entering and exiting fullscreen mode.                                          |
+
+<h2>Keyboard shortcuts</h2>
+
+The `VideoPlayer` component supports the following keyboard shortcuts for controlling video playback:
+
+- `Space` or `k`: Toggles between playing and pausing the video.
+- `m`: Toggles between muting and unmuting the video.
+- `f`: Toggles between entering and exiting fullscreen mode.
+- `c`: Toggles closed captions on and off.
+- `ArrowLeft`: Seeks the video backward by 5 seconds.
+- `ArrowRight: Seeks the video forward by 5 seconds.
+- `j`: Seeks the video backward by 10 seconds.
+- `l`: Seeks the video forward by 10 seconds.
+- `ArrowUp` or `ArrowDown`: Increases or decreases the volume by 10%.
```

Comment:
qq about this one... is this okay from accessibility perspective? Some users may use up/down keys to scroll the page and this overwrites that default behaviour. Were a11y team okay with us adding these particular shortcuts?

Reply:
This is a good question and I went back and forth on this based on reading I'd done.

I think for now I'm going to disable the keyboard interaction until I've had a discussion with the a11y team

Reply:
Resolved in 9944caca8bae4068b0ed3b6d8b2ced8b5308c58b